LaunchSpatial.bat

```diff
@@ -8,6 +8,6 @@ spatial worker build build-config
 rem To launch a local Spatial deployment with a different launch configuration, change the default_launch.json argument to the name of your configuration file, 
 rem for example one_worker_test.json or two_worker_test.json. Similarly, you can specify a different snapshot than the default one by replacing the value of 
 rem the --snapshot argument with the path to the snapshot you want to launch with.
-spatial local launch default_launch.json --snapshot=snapshots/default.snapshot --runtime_version 0.4.0
```

Comment:
yowzers

Reply:
hah this isn't as bad as it looks, squid went from 0.5 straight to 15.0